src/imitation/algorithms/mce_irl.py

```diff
@@ -45,7 +56,7 @@ def mce_partition_fh(
         \pi is a 3d array, indexed \pi[t,s,a].
     """
     # shorthand
-    horizon = env.horizon
+    horizon = int(env.horizon)
```

Comment:
Can we add a TODO here so we remember to fix this before merge (if the plan is to fix it in a seals PR first), or open a GH issue to track it (if we want to merge this PR before fixing it upstream)?

Reply:
I've opened a PR and added an extra check that should still be relevant even once the upstream has been changed, but can still add an extra comment if that would be helpful